bazel/bazel.go

```diff
@@ -63,17 +63,43 @@ func bazelNpmPath(ibazelBinPath string) (string, error) {
 	return "", errors.New("bazel binary not found in @bazel/bazel package")
 }
 
+// bazeliskNpmPath looks up a relative path to a binary from @bazel/bazelisk
+// This is used as an alternate resolution when no bazel binary is in the $PATH
+// When running from the @bazel/ibazel npm package, our binary is
+// /DIR/node_modules/@bazel/ibazel/bin/darwin_amd64/ibazel
+// We can find bazelisk in
+// /DIR/node_modules/@bazel/bazelisk/bazelizk-darwin_amd64
+func bazeliskNpmPath(ibazelBinPath string) (string, error) {
```

Comment:
Could I have you add a test for this? It doesn't depend on any system calls or anything so a [table driven test](https://github.com/golang/go/wiki/TableDrivenTests) would be sufficient.

Reply:
Sorry, added these earlier but couldn't figure out why I was getting a segfault, but fixed now.

Fortunately they caught me missing the `.exe` extension for Windows. I'm not 100% comfortable with how I'm doing the platform detection though.